modal/gpu.py

```diff
@@ -195,19 +197,16 @@ def _parse_gpu_config(value: GPU_T) -> Optional[_GPUConfig]:
             except ValueError:
                 raise InvalidError(f"Invalid GPU count: {count_str}. Value must be an integer.")
 
-        if value.lower() == "a100-20g":
-            return A100(size="20GB", count=count)  # Triggers unsupported error underneath.
-        elif value.lower() == "a100-40gb":
-            return A100(size="40GB", count=count)
-        elif value.lower() == "a100-80gb":
-            return A100(size="80GB", count=count)
-        elif value.lower() not in STRING_TO_GPU_CONFIG:
+        if value.lower() == "a100-40gb":
+            gpu_type = "A100-40GB"
```

Comment:
Any reason not to just have a many-to-one map in `STRING_TO_GPU_TYPE`?

Reply:
Oh because you don't want to present it as a separate option when you error out I gather.

Reply:
yeah i guess we could, no strong feelings